src/http-response-message.js

```diff
@@ -53,9 +51,9 @@ export class HttpResponseMessage {
     }catch(e){
       if(this.isSuccess){
         throw e;
+      } else {
```

Comment:
The else block is unnecessary and IMO slightly less clear than the previous version of this code.